packages/components/filters/src/filters.tsx

```diff
@@ -6,6 +8,14 @@ export type TFiltersProps = {
 };
 
 function Filters(props: TFiltersProps) {
-  return <div>{props.label}</div>;
+  return (
+    <>
+      <div>{props.label}</div>
+      <ul style={{ display: 'flex', gap: '8px' }}>
```

Comment:
do you think we should use spacing20 here? 

```suggestion
      <ul style={{ display: 'flex', gap: designTokens.spacing20 }}>
```